src/Models/ShallowWaterModels/solution_and_tracer_tendencies.jl

```diff
@@ -62,8 +62,24 @@ Compute the tendency for the height, h.
                                      forcings,
                                      clock)
 
+    U = 1
+
+    flux = zeros(grid.Nx+1, grid.Ny, grid.Nz)
+    
+    for i in 1:grid.Nx+1, j in 1:grid.Ny, k in 1:grid.Nz
+        
+        flux[i, j, k] = advective_tracer_flux_x(i, j, k, grid, advection, U, solution.h)
+        
+    end
```

Comment:
@francispoulin you don't want this loop in the tendency function. I think you just want

```julia
return advective_tracer_flux_x(i, j, k, grid, advection, U, solution.h)
```

Reply:
Agreed.  I will make that change today and update the PR